app/views/publish_interface/providers/visas/edit.html.erb

```diff
@@ -0,0 +1,51 @@
+<% page_title = "Visa sponsorship" %>
+<% content_for :page_title, title_with_error_prefix(page_title, @provider_visa_form.errors.present?) %>
+
+<%# content_for :before_content do %>
+  <%#= govuk_back_link_to(details_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)) %>
+<%# end %>
```

Comment:
Can uncomment this and update the path helper, now that the details page is in master.